cpp/src/io/parquet/page_delta_decode.cu

```diff
@@ -641,6 +666,20 @@ CUDF_KERNEL void __launch_bounds__(decode_block_size)
 
   bool const has_repetition = s->col.max_level[level_type::REPETITION] > 0;
 
+  // Exit early if the page is invalid
+  // MH: What to do if `has_repetition` is true?
```

Comment:
Hi @nvdbaranec,
Do you think we need to any special handling for lists (`has_repetiion` == true) in this kernel. I only see it being used to set the `dst` address. Same question for a couple other kernels.